src/services.ts

```diff
@@ -1,5 +1,14 @@
 /* eslint-disable no-useless-escape */
-export default {
+interface Service {
```

Comment:
it repeats the interface from index.ts. You can move types to a separate file and depend on it from both files